packages/photon/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "photon",
+	"version": "2.0.1",
+	"description": "JavaScript library for the WordPress.com Photon image manipulation service",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
```

Comment:
should add `sideEffects: false`

Reply:
4f03cbd4743db2f8f83a6745d6d9adff634df87e